### ISSUE TITLE
revert back to commit f1f3

### ISSUE THREAD

**Issue by groenborg** [Praqma/pretested-integration-plugin#79]
The branch `pipeline-compliant` have been branched from the commit f1f33822b878c93f3748d3d4f0915422855b32a5  We have added all changes until commit 56e87262ead8be9e7f5a1edb881222ced7aec518 on master. We will be reverting master all the way back to  f1f33822b878c93f3748d3d4f0915422855b32a5 to merge in the changes from our branch. We do the revert because we do not want to make a huge force merge with all our changes. To keep the history we simply revert and insert our changes.

**Comment by buep** [comment]
I can add....

And the reason while we did the `pipeline-compliant` branch was because master was a mess of a large number of not working commit that have been failing our pipeline for month.

So on the `pipeline-compliant` branch we took in changes one by one, and only pieces we needed and tried to get tests working along the way so we're could easier get to a working state.

So now our branch is the working one.

We find reverting to be more traceable than two other options considered: 1. rename branches, replacing master.  2. merge --use-ours style.
